src/main/features/response/tasks/howMuchPaidClaimantTask.ts

```diff
@@ -0,0 +1,7 @@
+import { ResponseDraft } from 'response/draft/responseDraft'
+
+export class HowMuchPaidClaimantTask {
+  static isCompleted (responseDraft: ResponseDraft): boolean {
+    return !!(responseDraft.howMuchPaidClaimant && responseDraft.howMuchPaidClaimant.option)
```

Comment:
Please use validator for validation. No point duplicating logic there. Check `YourDefenceTask.isCompleted` for reference.